test/testvarid.cpp

```diff
@@ -877,7 +877,7 @@ class TestVarID : public TestFixture {
     void varid36() { // ticket #2980 (segmentation fault)
         const char code[] ="#elif A\n"
                             "A,a<b<x0\n";
-        tokenize(code);
+        (void)tokenize(code);
```

Comment:
These could also be written as `ASSERT_NO_THROW()` and my changes here are as inconsistent as the existing code.

Reply:
I think there are also some unnecessary return values in the test code which could also be cleaned up in a follow-up.

Reply:
I filed https://trac.cppcheck.net/ticket/13006 about detecting functions which always have their result ignored.